ref/build.sh

```diff
@@ -151,5 +151,7 @@ make -j4 VERBOSE=1
 export OMP_NUM_THREADS=4
 export OMP_PLACES=cores
 export OMP_PROC_BIND=close
+ulimit -s 12000000 
```

Comment:
Since this line doesn't do anything, I say remove it.  Leaving it there, gives the reader the impression that it's needed for something when, in fact, it can't possibly help.